libs/expo/betterangels/src/lib/screens/AddEditClient/AddEditClient.graphql

```diff
@@ -32,6 +32,8 @@ query GetClientProfile($id: ID!) {
       physicalDescription
       displayCaseManager
       maritalStatus
+      mailingAddress
```

Comment:
@Davit-BetterAngels 
Quick question. 

When I found the address I needed to enter, only the street address display in the text box excluding the city and state that show up in the suggestion. We can't add the city and state in the textbox? Is this the question I should ask @kimchoi-jjiggae ?

Reply:
@shiozakiyuko we should have the full address, including zipcode. Now that I think about it, we should also have apt/unit. But the google api doesn't provide that, so I'm not sure what the best solution is. I think our only option atm is having users manually update the address _after_ selecting from the autocomplete options

Reply:
@shiozakiyuko there are APIs that do full mailing address, down to the unit. Maybe something to consider long term, once the app has been adopted by a few orgs